module_4/src/com/epam/aggregationandcomposition/entity/lexemes/Readable.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.aggregationandcomposition.entity.lexemes;
+
+import com.epam.aggregationandcomposition.exceptions.LexemeLogicalException;
+
+public interface Readable {
```

Comment:
Let's name it simplier, e.g. Lexeme, because Readable sounds a bit strange
```suggestion
public interface Lexeme {
```